ding/framework/wrapper/step_timer.py

```diff
@@ -37,7 +37,7 @@ def executor(ctx):
                 time_cost = time.time() - start_time
             self.records[step_id].append(time_cost * 1000)
             if ctx.total_step % self.print_per_step == 0:
-                print(
+                logging.warning(
```

Comment:
why use `warning`, I think `info` level can be better

Reply:
I think using warnings would ensure that these logs are displayed in the terminal, regardless of whether the base log level is an information or a warning. If someone doesn't need this information, he can just delete the step timer.